form/form.go

```diff
@@ -357,30 +378,28 @@ func makeStructEncoder(t reflect.Type) *structEncoder {
 			continue
 		}
 
+		fldTyp := reflectField.Type
```

Comment:
It looks like "typ" for types is the abbreviation used in Go core's source code too. Nice!